dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePdWorkshopSurvey < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_workshop_surveys do |t|
+      t.references :pd_enrollment, null: false, index: true, unique: true
```

Comment:
This misleadingly doesn't actually create a unique index. See below.

In my experience, I've had to add the index separately if I want it to be unique. Here's an [example](https://github.com/code-dot-org/code-dot-org/blob/0b5917fffa6b0c9f8521517ffe8ce5a04f0ca6b3/dashboard/db/migrate/20170407201709_create_pd_workshop_material_orders.rb#L27-L27)